torchci/pages/api/drci/drci.ts

```diff
@@ -867,15 +913,28 @@ export async function reorganizeWorkflows(
         headShaTimestamps.set(workflow.head_sha, headShaTimestamp);
       }
 
-      workflowsByPR.set(pr_number, {
-        pr_number: pr_number,
+      let prTitle = "";
+      let prBody = "";
+      let prShas: { sha: string; title: string }[] = [];
+      if (octokit) {
+        const prData = await fetchPR(owner, repo, `${prNumber}`, octokit);
```

Comment:
should we be worried about rate limits for this?

Reply:
Per our chat, this will incur the cost of 2 GitHub queries per PR

Reply:
I guess I will follow up with an alternative to query from Rockset if this turns out to be an issue.  I'm not quite sure if the pull_request Rockset table is always up to date.  Maybe there is a legit reason we are using GitHub query to get the PR info instead.